### PR TITLE
ownCloud 9.0 rework.

### PR DESCRIPTION
**Pull request by ypid**
I will test this a bit before merging as it includes a few bigger changes and a lot of improvements. After that, I have one feature PR ready to be merged later and then I guess we can make a solid 0.3.0 release :wink:

This PR depends on https://github.com/debops/ansible-nginx/pull/122 for ownCloud to not complain about any warnings.

### CODE REVIEW

**Comment by ypid** [comment]
I will fix the test build later. Some Ubuntu compatibility thing. I guess I will need a Ubuntu test instance now too.
@drybjed Are you OK with the maintainer switch?

**Comment by drybjed** [comment]
@ypid Yes, I'm fine with maintainer switch. Nice work :+1:

**Comment by drybjed** [comment]
Nice work. :+1:

File: .travis.yml [hunk 1]
```diff
@@ -12,5 +14,6 @@ install: True
 
 script:
   - 'git clone --depth 1 https://github.com/nickjj/rolespec'
-  - 'cd rolespec ; bin/rolespec -r https://github.com/debops/test-suite'
+  # - 'cd rolespec ; bin/rolespec -r https://github.com/debops/test-suite'
+  - 'cd rolespec ; bin/rolespec -r https://github.com/ypid/test-suite-ypid'
```
Comment (ypid) [inline .travis.yml:18]: Will be fixed later …

**Pull request merged**